src/SFA.DAS.RoATPService.Application/AuditLogFieldComparison.cs

```diff
@@ -27,6 +27,25 @@ public async Task<IEnumerable<AuditLogEntry>> BuildListOfFieldsChanged(Organisat
                 }
             );
             ComparisonResult comparisonResult = organisationComparison.Compare(originalOrganisation, updatedOrganisation);
+            var updatedAt = updatedOrganisation.UpdatedAt;
+            if (!updatedAt.HasValue)
+            {
+                updatedAt = DateTime.Now;
+            }
+
```

Comment:
simplify above to
      var updatedAt = updatedOrganisation.UpdatedAt ?? DateTime.Now ;